From 6125217f249418d9d3637b4c03f5d8642dd8e7f2 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 7 Jan 2018 18:12:59 +0200
Subject: [PATCH 2195/2241] net: mvpp2x: print gmac and phy info in port-probe

Print-info-level gop-gmac-id, phy-mode and phy-speed
in port_probe.

Change-Id: I150f2c70f5d808ee4605598b6141ea4f893695c1
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48616
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index 352e441..c3845b8 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -4757,9 +4757,8 @@ static int mv_pp2_init_emac_data(struct mv_pp2x_port *port,
 		}
 	}
 	port->mac_data.phy_mode = phy_mode;
-	pr_debug("gop_mac(%d), phy_mode(%d) (%s)\n", id,  phy_mode,
-		 phy_modes(phy_mode));
-	pr_debug("gop_mac(%d), phy_speed(%d)\n", id,  port->mac_data.speed);
+	pr_info("gop_mac(%d), phy_mode(%d)=%s phy-speed=%d\n", id,  phy_mode,
+		phy_modes(phy_mode), port->mac_data.speed);
 
 	phy_node = of_parse_phandle(emac_node, "phy", 0);
 	if (phy_node) {
-- 
2.7.4

